vllm/entrypoints/openai/api_server.py

```diff
@@ -112,11 +113,23 @@ async def build_async_engine_client(args) -> AsyncIterator[AsyncEngineClient]:
                     rpc_path)
 
         # Start RPCServer in separate process (holds the AsyncLLMEngine).
-        rpc_server_process = Process(target=run_rpc_server,
-                                     args=(engine_args,
-                                           UsageContext.OPENAI_API_SERVER,
-                                           rpc_path))
-        rpc_server_process.start()
+        # NOTE: we don't use multiprocessing to avoid fork/spawn issues.
+        # subprocess can give us a clean process with no shared state.
+        rpc_server_process = subprocess.Popen([
+            sys.executable,
+            "-m",
+            "vllm.entrypoints.openai.rpc.server",
+        ],
+                                              stdin=subprocess.PIPE,
+                                              stdout=sys.stdout,
+                                              stderr=sys.stderr)
+        logger.info("Started engine process with PID %d",
```

Comment:
log the engine process pid, to debug the long existing flaky tests for pipeline parallel.